### DISCUSSION TITLE
How to build from sources

### DISCUSSION THREAD

**Discussion by wrexroad** [gitpod-io/openvscode-server#295] [Q&A] [answered]
How to I was wondering if there is any updated information about building from sources. I have read this: https://github.com/gitpod-io/openvscode-server/blob/docs/development.md where it says to start a Gitpod instance and run `yarn gulp server-min` to create a package but I get the error "Task never defined". I looked at the task list and found a similarly named task 'vscode-min' but it doesn't look like its creating the distribution package. Can anyone tell me what I'm doing wrong?

**Comment by jeanp413** [comment, marked as answer]
Hi @wrexroad, sorry for the outdated docs, to build from sources you need to run `yarn gulp vscode-reh-web-linux-x64-min`

**Reply by btwiuse** [reply]
Might be a stupid question, but what does `reh` mean? I can't find any explanation in the docs.

**Reply by jeanp413** [reply]
Hi @btwiuse, reh => remote extension host

**Comment by wrexroad** [comment]
Perfect, thanks so much!
